bot/constants.py

```diff
@@ -257,7 +257,8 @@ class Tokens(NamedTuple):
     youtube = environ.get("YOUTUBE_API_KEY")
     tmdb = environ.get("TMDB_API_KEY")
     nasa = environ.get("NASA_API_KEY")
-    igdb = environ.get("IGDB_API_KEY")
+    igdb_client_id = environ.get("IGDB_CLIENT_ID")
+    igdb_client_secret = environ.get("IGDB_CLIENT_SECRET")
```

Comment:
https://pythondiscord.com/pages/contributing/sir-lancebot/sir-lancebot-env-var-reference/#tokensapis
Will need to be updated to reflect these env var changes.
The current link in the reference table correctly explains how to get them.

The old env var is not used anywhere now, so it has been deleted.
These 2 new env vars will need to be available within the bot's environment when this goes live (preferably before).